bleak/exc.py

```diff
@@ -16,18 +16,34 @@ class BleakDotNetTaskError(BleakError):
 class BleakDBusError(BleakError):
     """Specialized exception type for D-Bus errors."""
 
-    def __init__(self, dbus_error: str):
+    def __init__(self, dbus_error: str, error_body: list = None):
         """
         Args:
             dbus_error (str): The D-Bus error, e.g. ``org.freedesktop.DBus.Error.UnknownObject``.
+            error_body (str): Optional body of the D-Bus error, sometimes containing error description or details.
         """
-        super().__init__(dbus_error)
+        super().__init__(dbus_error, error_body)
```

Comment:
Would it make sense to unpack `error_body` here? Then the exception `args` index will just be the body index plus 1.

    super().__init__(dbus_error, *error_body)

Reply:
Now that `error_body` is not optional, indeed :+1: